packages/example-forum/lib/server/posts/callbacks/other.js

```diff
@@ -21,6 +21,10 @@ registerSetting('forum.trackClickEvents', true, 'Track clicks to posts pages');
  * @summary Increment the user's post count
  */
 function PostsNewIncrementPostCount(post) {
+  if (post.draft === true) {
```

Comment:
Is it OK to edit things in "example-forum", or are we leaving that unchanged so that we can pull in updates?